### PR TITLE
[8.0] remove update stock move line when create invoice

### PR DESCRIPTION
**Pull request by sergiocorato**
WIP test todo

### CODE REVIEW

**Comment by rousseldenis** [comment]
@sergiocorato Could you maybe add a test to enlight the change ?

**Comment by rousseldenis** [comment]
@sergiocorato Could you update modules version and add some tests ?

**Comment by sergiocorato** [comment]
> @sergiocorato Could you update modules version and add some tests ?

@rousseldenis ok version, as it is an improvement on execution time, add some time-execution-test?

File: stock_picking_invoice_link/models/stock_move.py [hunk 1]
```diff
@@ -28,6 +28,7 @@ def _compute_invoice_line_id(self):
     def _create_invoice_line_from_vals(self, move, invoice_line_vals):
         inv_line_id = super(StockMove, self)._create_invoice_line_from_vals(
             move, invoice_line_vals)
-        move.invoice_line_ids = [(4, inv_line_id)]
+        move.with_context(skip_update_line_ids=True).invoice_line_ids = [
```
Comment (rousseldenis) [inline stock_picking_invoice_link/models/stock_move.py:31]: @sergiocorato Is this 'skip_update_line_ids') present in core (I could not find it)?

Comment (sergiocorato) [reply stock_picking_invoice_link/models/stock_move.py:31]: @rousseldenis no, it is in `stock-logistick-workflow` repo, in module `stock_picking_package_preparation_line`

**Comment by rousseldenis** [comment]
@sergiocorato Do you plan to add tests or is this ready ?

**Comment by sergiocorato** [comment]
> @sergiocorato Do you plan to add tests or is this ready ?

@rousseldenis I think we can proceed as-is

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**